docs/logs/correlation/README.md

```diff
@@ -1,7 +1,20 @@
 # Correlate Logs with Traces
 
-Logs from `ILogger` can be correlated with distributed tracing by enriching each
-log entry with the `TraceId` and `SpanId` from the enclosing `Activity`.
+Starting from `ILogger` version `5.0`, logs can be correlated with distributed
```

Comment:
We typically refer to it as `ILogger`, but should we instead say, starting from `Microsoft.Extensions.logging` version 5.0?

(Also can 3.1  app get this feature by upgrading just this package to 5.0? (Similar to DiagnosticSource)